package.json

```diff
@@ -4,8 +4,9 @@
   "description": "Book-A-Meal is an application that allows customers to make food orders and helps the food",
   "main": "server.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1",
-    "start-dev": "nodemon --exec babel-node src/server.js"
+    "start-dev": "nodemon --exec babel-node src/server.js",
```

Comment:
I can't see a start script ??

Reply:
thats is what the start-dev does

Reply:
Heroku looks for `start` script and not `start-dev`. You can't used `nodemon` into production @Mohamilr

Reply:
done sir